src/CardPaymentModal.svelte

```diff
@@ -40,11 +41,19 @@
       const options = get(optionsStore)
 
       try {
-        options.approveCardPayment(cardPaymentData.paymentId);
+        options.approveCardPayment(cardPaymentData.paymentId, cardPaymentData.currency);
         success = true;
+
+        if (successUrl) {
+          window.location.href = successUrl
```

Comment:
```suggestion
          location.href = successUrl
```
Does it work without `window`?